integration/test_resources/help/report-definition_list

```diff
@@ -7,7 +7,8 @@ Aliases:
   list, ls
 
 Flags:
-  -h, --help   help for list
+  -h, --help             help for list
+      --subtype string   filter report definitions by subtype. 'Aws', 'GCP' or 'Azure'
```

Comment:
The help for this command gives the following
`--subtype string   filter report definitions by subtype. 'Aws', 'GCP' or 'Azure'`

This does not work with Aws, Gcp they need to match the full AWS, GCP, Azure etc. Perhaps make the user input choice uppercase for the check for Gcp and Aws?

Also if the user inputs `lacework report-definition list --subtype AZURE `then the error is 
`ERROR 'AZURE' is not valid. Report definitions subtype can be [AWS Azure GCP]